lmcommon/configuration/configuration.py

```
@@ -61,6 +61,14 @@ def find_default_config() -> str:
             # Load default file out of python package
             return os.path.join(resource_filename("lmcommon", "configuration/config"), "labmanager.yaml.default")
 
+    @property
+    def host_cuda_version(self) -> Optional[str]:
+        try:
+            return self.config['container'].get('cuda_version')
+        except KeyError:
+            logger.warning('Missing cuda_version in configuration')
```

since using `.get()` this should only throw a key error if the "container" section is missing, right?